### DISCUSSION TITLE
Scheduled payment requests

### DISCUSSION THREAD

**Discussion by SpeedyCoinzales** [btcpayserver/btcpayserver#3494] [Ideas / Feature Requests]
Hi, 
use case: I've service contract with a customer and need to settle periodical payment requests. Every week, moth, quarter or year, for example.
solution: additional parameters to set an automatic workflow to create a payment request + email notification of the customer. 
For this feature, the email notification option should be more customized.

Many Thanks for your great work

**Comment by pavlenex** [comment]
That makes sense, in a way it's similar t this issue https://github.com/btcpayserver/btcpayserver/issues/461 it sounds like a neat workflow tbh.
